src/androidMain/kotlin/com/mutualmobile/libphonenumber/PhoneUtil.kt

```diff
@@ -0,0 +1,63 @@
+package com.mutualmobile.libphonenumber
+
+actual object PhoneUtil {
+
+    @Volatile
+    private var instance: PhoneNumberUtil? = null
+
+    actual fun getInstance(context: ApplicationContext) {
```

Comment:
Should this method be renamed to something like `initialize` to signify that this method is just placing a value on the `instance` variable?